### PR TITLE
Raincatch 1274 - Working with the WFM templates on RHMAP

### PR DESCRIPTION
**Pull request by jstaffor**
1) I applied most of the content into a procedure as this is exactly what it is
2) It is only being rendered in downstream
2.1) I've copied the User Stories into the rhmap dir and overwritten the sym links
3) I've removed some downstream logic from upstream-1 User Stories as it is no longer needed as the User Stories are no longer sym links in downstream, I now have copies ( reread point 2)

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
+[id='{context}-pro-working-with-templates-on-rhmap']
+= Working with {Feature} Templates in {ProductShortName}
+
+This section includes:
+
+. xref:{context}-creating-a-new-wfm-project-in-rhmap[Creating a new {Feature} Project in {ProductShortName}]
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc:6]: See project "DeleteMeWhenever_IAmNotImportant4" on https://testing.zeta.feedhenry.com/#projects/ as to why I have used these names

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+NOTE: If you makes changes to the WFM Demo Cloud App, ensure you redeployed the WFM Demo Cloud App
+
+[id='{context}-setting-up-and-deploying-the-wfm-demo-portal-app']
+[discrete]
+== Setting Up and Deploying the WFM Demo Portal App
+
+. On the _WFM Project_ homepage, click on the *WFM Demo Portal App*
+. In the left side navigation panel, click on *Deploy* to navigate to the deployment screen
+. On the deployment screen, select *Node.js - 6.1.9* from the *Choose a Runtime* list
+. To deploy the _WFM Demo Portal App_, click on the *Deploy Cloud App*
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc]: Is this a bug on zeta, should it say "*Deploy Portal App*" on the button?

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+[discrete]
+== Setting Up and Deploying the WFM Demo Cloud App
+
+. On the _WFM Project_ homepage, click on the *WFM Demo Cloud App*
+. In the left side navigation panel, click on *Data Browser*
+. If the database requires an update, select *Upgrade database*
+.. A warning message appears, click on *Upgrade Now* to proceed
+. Once the upgrade is complete, click *Next*
+. Navigate to the deployment screen by clicking *Deploy*
+. On the deployment screen, select *Node.js - 6.1.9* from the *Choose a Runtime* list
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc]: There is no runtime 6.1.9 on zeta, only 6.9.1. I hope that I was not testing on an old version as I asked for the latest representation of the application "I am wanting to test https://github.com/feedhenry-raincatcher/raincatcher-docs/pull/72/files against the latest representation of the application"

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+[id='{context}-setting-up-and-building-the-wfm-demo-mobile-app']
+[discrete]
+== Setting Up and Building the WFM Demo Mobile App
+
+. On the _WFM Project_ homepage, click on the *WFM Demo Mobile App*
+. In the left side navigation panel, click on *Build* to navigate to the build screen
+.. To build a mobile binary, see link:https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform/4.5/html-single/local_development_guide/#building-an-app-binary[Building an App Binary]
+.. To build a mobile binary on the platform, see link:https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform/4.5/html-single/getting_started/#run-the-client-app-on-a-mobile-device[Run the Client App on a Mobile Device]
+
+NOTE: If you makes changes to the WFM Demo Mobile App, ensure you apply them to the _www_ directory using the `grunt` command
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc]: "using the `grunt` command" - this is very vague, will a developer know what to do here?

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc (hunk 3 above)
Comment (austincunningham) [reply docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc]: good catch

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc (hunk 2 above)
Comment (austincunningham) [reply docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc]: believe it is expected any web app is a cloud app

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc (hunk 4 above)
Comment (austincunningham) [reply docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc]: `grunt` cli build tool 

```
grunt
```

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc (hunk 1 above)
Comment (wtrocki) [reply docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc:6]: @jstaffor - this names were changed.  Templates on zeta are old.
Here are the actual names (but we can also change them if other suggested)
https://github.com/feedhenry/fh-template-apps/blob/master/global.json#L473-L514

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc (hunk 3 above)
Comment (wtrocki) [reply docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc]: As from email we should use Node 6 LTS  and this should be variable.

**Comment by wtrocki** [comment]
@jstaffor This looks good. I think we will need to review entire docs again before release and catch if we using demo/reference applications in some examples and get them unified.

**Review by wtrocki — APPROVED** [review]

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc (hunk 1 above)
Comment (jstaffor) [reply docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc:6]: I will update these to:

 - WFM Mobile App
 - WFM Portal App
 - WFM Cloud App

and apply in a general review of repoo

File: docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc (hunk 3 above)
Comment (jstaffor) [reply docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc]: I'll update this and apply in a general review of repoo

**Pull request merged**